src/gui/wizard/owncloudsetuppage.cpp

```diff
@@ -63,8 +63,7 @@ OwncloudSetupPage::OwncloudSetupPage(QWidget *parent)
             const auto serverObject = serverJson.toObject();
             const auto serverName = serverObject.value("name").toString();
             const auto serverUrl = serverObject.value("url").toString();
-            const auto serverDisplayString = QStringLiteral("%1 (%2)").arg(serverName, serverUrl);
-            _ui.comboBox->addItem(serverDisplayString, serverUrl);
+            _ui.comboBox->addItem(serverName, serverUrl);
```

Comment:
just curious - why this change?

Reply:
- `QStringLiteral()` constructs the `QString` object during compilation, which avoids the runtime overhead of constructing `QString` objects.